src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -99,4 +99,9 @@
         public static final int XBOX_PORT = 0;
         public static final double RUMBLE_INTERMISSION_TIME = 0.15;
     }
+
+    public static class ShooterConstants {
+        public static final TrigonTalonFX RIGHT_MOTOR = ShooterComponents.RIGHT_MOTOR;
+        public static final TrigonTalonFX LEFT_MOTOR = ShooterComponents.LEFT_MOTOR;
```

Comment:
left above right just because of the alphabet

Reply:
refer to above

Reply:
fix